bin/ember-template-lint.js

```diff
@@ -347,7 +347,10 @@ async function run() {
       config,
       rule: options.rule,
       allowInlineConfig: !options.noInlineConfig,
-      console: options.quiet || options.json ? NOOP_CONSOLE : console,
+      console:
+        options.quiet || options.json || ['sarif', 'json'].includes(options.format)
```

Comment:
Can you hoist this to a variable or two above? This is getting somewhat hard to reason about...

Reply:
Some might say it's unreasonable :)